node/src/Commands.ts

```diff
@@ -3459,3 +3459,75 @@ export function createBZPopMin(
 ): command_request.Command {
     return createCommand(RequestType.BZPopMin, [...keys, timeout.toString()]);
 }
+
+/**
+ * Optional arguments to getex command.
+ */
+export type GetExOptions = {
+    expiry?: /**
```

Comment:
If this is the only member of the type then it shouldn't be optional
```suggestion
    expiry: /**
```

Reply:
good catch